mikrotik/resource_bridge_port_test.go

```diff
@@ -68,6 +66,23 @@ func testAccBridgePortExists(resourceID string, record *client.BridgePort) resou
 }
 
 func testAccCheckBridgePortDestroy(s *terraform.State) error {
+	c := client.NewClient(client.GetConfigFromEnv())
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "mikrotik_bridge_port" {
+			continue
+		}
+
+		remoteRecord, err := c.FindBridgePort(rs.Primary.ID)
```

Comment:
Once we have migrated to the solution in #116, I hope we can remove these `testAccChec*Destroy` functions for a generic one.